apps/admin-page/src/app/pages/account/account.component.ts

```diff
@@ -0,0 +1,139 @@
+import { HttpErrorResponse } from '@angular/common/http';
+import { Component } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { IUser } from '@interfaces';
+import { UserRoles } from '@models';
+import { UserService } from '../../services/user/user.service';
+import { MatDialog } from '@angular/material/dialog';
+import { ChangePasswordDialogComponent } from '../../components/change-password-dialog/change-password-dialog.component';
+
+@Component({
+  selector: 'webstore-account',
+  templateUrl: './account.component.html',
+  styleUrls: ['./account.component.css'],
+})
+export class AccountComponent {
+  public user?: IUser;
+  public accountForm: FormGroup;
+  public isLoading = false;
+  public isUpdatingInfo = false;
+  public panelOpenState = false;
+
+  /**
+   * Gets the current user.
+   * Initiates the account form.
+   *
+   * @param userService
+   * @param snackBar
+   * @param dialog
+   */
+  constructor(private userService: UserService, private snackBar: MatSnackBar, public dialog: MatDialog) {
+    this.isLoading = true;
+    this.userService.getCurrentUser().subscribe(
+      (user: IUser) => {
+        this.user = user;
+        this.updateForm();
+        this.isLoading = false;
+      },
+      (err: HttpErrorResponse) => {
+        console.log(err.message);
```

Comment:
You should display some kind of error to the user if the request fails. Either a snackbar or a HTML element

Reply:
When logging errors, you should use `console.warn` instead of log for better readability
```suggestion
        console.warn(err.message);
```